src/main/java/frc/robot/constants/RobotConfig.java

```diff
@@ -3,14 +3,130 @@
+    public static final String kTopFlywheelMinOutputKey = "Top Flywheel Minimum output";
+    public static final String kTopFlywheelMaxOutputKey = "Top Flywheel Maximum output";
+
+    // Roller Default Speed
+    public static final double kRollerDefaultSpeed = -0.4;
+    // Flywheel default speed
+    public static final double kFlywheelDefaultRPM = 1000;
+    // Shield Retracted position
+    public static final double kShieldRetractedPosition = 15;
+    // Timeout time (in seconds)
+    public static final double kRunIntakeTimeoutTime = 0;
+    public static final double kShieldExtendedPosition = 85; // TODO get correct value
+    // Speaker height
+    public static final double SpeakerHeight = 1.9812;
+    public static final double AmpHeight = .46;
+    public static final double ShooterHeight = 0.28575;
+    public static final double TrapHeight = 1;
+
+    public static final double SpeakerBillLength = 0.6604;
+
+    public static final int shootButton = 1;
+
+    public static final double kDefaultFlywheelRPM = 1000;
+
+    public static final double kShooterStowAngle = 0;
+
+    public static final long kReleaseTime = 5000;
+    public static final long kShieldTime = 2; // seconds
+    public static final double kShieldDefaultSpeed = 0.5;
+    public static final Measure<Velocity<Angle>> kFlywheelError = Units.RPM.of(1);
+    public static final Measure<Angle> kAngleError = Units.Radians.of(0.5 * Math.PI / 180);
+    public static final Measure<Angle> kSpeakerAngle = Units.Radians.of(75 * Math.PI / 180);
+    public static final Measure<Angle> kAmpAngle = Units.Radians.of(109 * Math.PI / 180);
+    public static final Measure<Angle> kTrapAngle = Units.Radians.of(105 * Math.PI / 180);
+    public static final Measure<Angle> kAdjustAmountDegrees = Units.Radians.of(0.5 * Math.PI / 180);
+
+    // TODO placeholders
+    public static final double ampVelocity = 1500; // rpm
+    public static final double trapVelocity = 2000; // rpm
+    public static final double speakerVelocity = 2500;
+  }
 
   public static class DriveConfig {
+    public static class TranslateConfig {
+      public static final String kPKey = "Vision Translate P";
+      public static final String kIKey = "Vision Translate I";
+      public static final String kDKey = "Vision Translate D";
+      public static final double kP = 0.0;
+      public static final double kI = 0.0;
+      public static final double kD = 0.0;
+      public static final double kTolerance = 1.0;
+      public static final double minIntegral = 0;
+      public static final double maxIntegral = 2;
+    }
+
+    public static class TurnConfig {
+      public static final String kPKey = "Vision Turn P";
+      public static final String kIKey = "Vision Turn I";
+      public static final String kDKey = "Vision Turn D";
+      public static final double kP = 0.0;
+      public static final double kI = 0.0;
+      public static final double kD = 0.0;
```

Comment:
Should the PID's have non-0 values?

Reply:
yep

Reply:
Anything related to vision is unused so 0 pid constants on those is probably fine.